### PR TITLE
deliver

### PR DESCRIPTION
**Pull request by TNThacker2015**

### CODE REVIEW

File: commands/private/deliver.cmd.js [hunk 1]
```diff
@@ -12,11 +12,11 @@ module.exports =
 		.setPermissions(canCook)
 		.setFunction(async(message, args, client) => {
 			if (message.channel.id !== deliveryChannel) return message.channel.send("You can only use this command in the delivery channel");
-			if (!args[1]) message.channel.send("You need to specify an order");
+			if (!args[0]) return message.channel.send("You need to specify an order");
 
 			const order = await Orders.findOne({ where: { id: args.shift() } });
 			const worker = await WorkerInfo.findOne({ where: { id: message.author.id } });
-
```
Comment (Yamboy1) [inline commands/private/deliver.cmd.js]: This is one of the two useful changes here...

File: commands/private/deliver.cmd.js [hunk 2]
```diff
@@ -12,11 +12,11 @@ module.exports =
 		.setPermissions(canCook)
 		.setFunction(async(message, args, client) => {
 			if (message.channel.id !== deliveryChannel) return message.channel.send("You can only use this command in the delivery channel");
-			if (!args[1]) message.channel.send("You need to specify an order");
```
Comment (Yamboy1) [inline commands/private/deliver.cmd.js]: The other of the two userful changes

**Comment by TNThacker2015** [comment]
the sequelize one's username thing was for in case someone left.

**Comment by Yamboy1** [comment]
Only the stuff that i've mentioned here is actually useful/needed:

**Comment by Yamboy1** [comment]
https://github.com/Yamboy1/discord-donuts/pull/44#discussion_r229135751
https://github.com/Yamboy1/discord-donuts/pull/44#discussion_r229135810

**Comment by Yamboy1** [comment]
not merging, but i have put some changes from here into another pr.

**Pull request closed without merging**